0005-incident_response.md

```diff
@@ -0,0 +1,52 @@
+---
+oip: "0005"
+title: Incident Response
+status: Draft
+type: Operations
+author: Jake Bunce <jake@omise.co>
+created: 2018-11-
+---
+
+# OIP-0005: Incident Response 
+
+## Summary
+
+Outages are common place when running services. Managing these incidents with a defined process helps to restore services faster and creates trust for the users.
+
+## What defines an incident?
+
+An incident is an event that affects normal operation of an OmiseGO service. This can be a service that is used internally or externally, such as an automation service or a Plasma blockchain. Services are measured to determine normal operation through service level indicators (SLIs), from which service level objectives (SLOs) can be applied.
+
+For example, we may give assurances to third party partners that an endpoint API on a Watcher instance is available for 99.999% of the time. And we may give an assurance that the same API returns a response within 300ms 99% of the time. If we're not meeting those service level objectives then an incident should be raised.
+
+## Priority Levels
+
+Priority levels are used to define the business impact to a technical service failure. Incidents can start at one level and be raised/lowered as the incident progresses.
+
+### P0
+
```

Comment:
I'd suggest that executive levels are informed when P0 incidents are reported.

Reply:
Areas of notification added in 3452b08